src/main/java/ay2021s1_cs2103_w16_3/finesse/storage/JsonAdaptedFrequentExpense.java

```diff
@@ -30,7 +30,7 @@
      */
     @JsonCreator
     public JsonAdaptedFrequentExpense(@JsonProperty("title") String title, @JsonProperty("amount") String amount,
-                              @JsonProperty("categories") List<JsonAdaptedCategory> categories) {
+                                          @JsonProperty("categories") List<JsonAdaptedCategory> categories) {
```

Comment:
```suggestion
            @JsonProperty("categories") List<JsonAdaptedCategory> categories) {
```
For some reason checkstyle accepts indentation level 8, though according to the textbook in this case it should be indentation level 12.